notebooks/ISS_Pipeline_-_Breast_-_1_FOV.py

```diff
@@ -76,49 +77,41 @@
 # EPY: END markdown
 
 # EPY: START code
-codebook = pd.read_csv('http://czi.starfish.data.public.s3-website-us-east-1.amazonaws.com/ISS/codebook.csv', dtype={'barcode': object})
+codebook = pd.read_csv('https://dmf0bdeheu4zf.cloudfront.net/20180606/ISS/codebook.csv', dtype={'barcode': object})
 codebook.head(20)
 # EPY: END code
 
 # EPY: START markdown
-# ## Filter and scale raw data 
+# ## Filter and scale raw data
 # 
-# Now apply the white top hat filter to both the spots image and the individual channels. White top had enhances white spots on a black background. 
+# Now apply the white top hat filter to both the spots image and the individual channels. White top had enhances white spots on a black background.
 # EPY: END markdown
 
 # EPY: START code
-from starfish.filters import white_top_hat 
+from starfish.pipeline.filter import Filter
```

Comment:
This import differs from the way we import the gaussian spot detector. Should we be consistent?

Reply:
Yes.  What's your preference?

Reply:
I think I prefer the way you've done it here, because it reinforces that all methods need to be imported in `__init__.py` (You'll notice that many currently are _not_ -- I didn't realize we needed to do that when I was changing some of them over, and so the CLI will not include them). 

.. but the API still works if you just incorporate the methods, so I think this is a more robust way to enforce API == CLI.

Reply:
I fixed the gaussian spot instance.